agent/src/softwarecontaineragent.h

```diff
@@ -235,5 +248,6 @@ class SoftwareContainerAgent : protected softwarecontainer::JSONParser
     SignalConnectionsHandler m_connections;
     bool m_shutdownContainers = true;
     std::vector<ContainerID> m_containerIdPool;
+    ConfigStore m_configStore = ConfigStore("");
```

Comment:
I agree, but couldn't get it to compile then. I will grab you to see if you can see a solution.